interface.dd

```diff
@@ -70,6 +70,17 @@ interface D
 }
 ------
 
+    $(P Inferfaces can have function templates in the members.
```

Comment:
typo: Inferfaces. Also the sentence is a bit stilted, use: " Interfaces are allowed to define function templates."